playwright/test/landing-pages.test.ts

```diff
@@ -52,18 +75,6 @@ test.describe('Top-level landing pages', () => {
     const content = await page.textContent('h1');
```

Comment:
[page.textContent](https://playwright.dev/docs/api/class-elementhandle#element-handle-text-content) is discouraged. Might be best to use `page.getByRole('heading', {...})` for this one and the ones above!

Reply:
But again I'm not super sure of the value of these tests (all the "renders with an accessible title" ones)

Reply:
Maybe! Apparently @kenoir [added those in](https://github.com/wellcomecollection/wellcomecollection.org/pull/6631) 3 years ago. I _feel_ like Check Url now covers those cases but might be good to confirm?